src/jsonapi-vuex.js

```diff
@@ -398,6 +398,35 @@ const getters = () => {
       }
       return result
     },
+    getRelated: (state, getters) => (data) => {
+      let parent
+      const [type, id, rel] = getTypeId(data)
+      if (type in state) {
+        if (hasProperty(state[type], id)) {
+          parent = state[type][id]
+          const relations = get(parent, [jvtag, 'relationships', rel], {})
+          let relationsData = {}
+          let data = {}
+          if (relations.data) {
+            if (!Array.isArray(relations.data)) {
+              data[relations.data.id] = getters.get(
+                [relations.data.type, relations.data.id].join('/')
+              )
+              Object.assign(relationsData, data)
+            } else {
+              for (let relation of relations.data) {
```

Comment:
Can this `if/else` block be made more DRY by always converting `relations.data` to an array at the start, and then only running the `else` block's code?

Reply:
This would indeed be so much simpler. I keep wondering why JS does not have standard functionality for this (if array then loop, if not treat as a single instance). I'll change it but I don't know if or how this will affect performance)(negligible in this case, but you never know what problems it could cause in the future).